Config/SVTSegmentOvFile.txt

```diff
@@ -2,14 +2,47 @@
 # SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #====================== Segment Override File ======================
-# One input frame have to be preceded by the number of regions
-# Each region consist of 6 numbers (X1,Y1,X2,Y2, data, mode) i.e (X1,Y1)top left and (X2,Y2)bottom right corner
-# Mode parameter have to be in range [1..5]. This allow user to specify what to override by the [data]
-#    1.   Set qp value for each region, [data] is clipped to [0:51]
-#    2.   Change qp value i.e increase or decrease by the [data] clipped to [-25:25]
-#    3.   Use this mode to modify not only quality but also blocks splitting, behaviour depends on parameter 1 and 2
-#    4.   Use this mode to modify blocks splitting i.e increase or decrease by the [data] clipped to [-25:25], negative value increase block division
-#    5.   Update first stage of quantization in range [-7;7] positive value reduce bitrate, negative increase
+# 1. Purpose/use case of this feature
+#    This feature is for very specific use-cases like surveillance systems, where dynamic objects such as pedestrians or vehicles are detected and tracked (encoded with higher quality) while the static background is encoded with worse quality.
+#    By encoding only the important/key areas with good quality, the resulting bitstream is smaller and still has high usefulness.
+# 2. Usage:
+# For each frame in the video, the user can specify regions with better or worse quality.
+# One input frame have to be preceded by the number of regions.
+# Each region consists of 6 numbers (X1, Y1, X2, Y2, data, mode) i.e (X1,Y1)is the top left and (X2,Y2) is the bottom right corner.
```

Comment:
Each region consists of 6 numbers (X1, Y1, X2, Y2, data, mode) i.e (X1,Y1) is the top left and (X2,Y2) is the bottom right corner.  Regions are allowed to overlap in the same frame.